src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    addBook() {},
+    removeBook() {},
```

Comment:
- [ ] Good job working on the "Bookstore: add reducers and actions" project, however, currently we are not able to `add` and `remove` boob. Kindly make sure you add a reducer that adds and a reducer that removes a book as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer,removes%20a%20book) 😃